apps/front-end/src/onest/scholarship/AutomatedForm.jsx

```
@@ -265,22 +205,22 @@ const AutomatedForm = () => {
       const data = await response.json();
       // Set state and open the modal
       if (data.responses.length) {
-        for (let i = 0; i < data.responses.length; i++) {
-          if (data.responses[i].hasOwnProperty("message")) {
+        for (const response of data.responses) {
+          if (response.hasOwnProperty("message")) {
```

Biome

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)